app/views/collections/index.html.erb

```diff
@@ -55,14 +58,14 @@
   <h3 class="landmark heading"><%= ts("List of Collections") %></h3>
   <ul class="collection picture index group">
     <% @collections.each do |collection| %>
-      <%= render :partial => "collection_blurb", :locals => {:collection => collection} %>
+      <%= render partial: "collection_blurb", locals: { collection: collection } %>
```

Comment:
The syntax here is pretty old. We actually don't need `partial` or `locals` here anymore -- it can just be `<%= render "collection_blurb", collection: collection %>`